.circleci/config.yml

```diff
@@ -1,30 +1,24 @@
 version: 2.1
 
 parameters:
+  resource_class:
+    type: string
+    default: "macos.x86.medium.gen2"
   xcode_version:
     type: string
-    default: "13.2.1"
+    default: "14.3.1"
```

Comment:
Should this be 15.4?

Reply:
We don't own Apollo code. And this is still quite a few versions behind but we can't update to it

Reply:
Apollo's CI also doesn't run on our fork